apps/test/unit/templates/studioHomepages/teacherHomepageV2/SectionCardBodyTest.tsx

```diff
@@ -0,0 +1,147 @@
+import {fireEvent, render, screen} from '@testing-library/react';
+import React from 'react';
+import {Provider} from 'react-redux';
+import {
+  createMemoryRouter,
+  createRoutesFromElements,
+  Outlet,
+  Route,
+  RouterProvider,
+  useLocation,
+} from 'react-router-dom';
+import {Store} from 'redux';
+
+import {getStore} from '@cdo/apps/redux';
+import {SectionCardBody} from '@cdo/apps/templates/studioHomepages/teacherHomepageV2/SectionCardBody';
+import {Section} from '@cdo/apps/templates/teacherDashboard/types/teacherSectionTypes';
+import {
+  TEACHER_NAVIGATION_SECTIONS_URL,
+  SPECIFIC_SECTION_BASE_URL,
+  TEACHER_NAVIGATION_PATHS,
+} from '@cdo/apps/templates/teacherNavigation/TeacherNavigationPaths';
+
+const LocationElement = () => {
+  const location = useLocation();
+  return <div>{location.pathname}</div>;
+};
+
+describe('SectionCardBody', () => {
+  const section: Section = {
```

Comment:
Seems like we are using this section object a bunch. Don't have to right now, but maybe a cleanup task would be moving these to a test util file that could accept prop overrides or something.